lib/irb/completion.rb

```diff
@@ -296,9 +296,9 @@ def self.retrieve_completion_data(input, bind: IRB.conf[:MAIN_CONTEXT].workspace
         sep = $2
         message = $3
 
-        gv = eval("global_variables", bind).collect{|m| m.to_s}.push("true", "false", "nil")
-        lv = eval("local_variables", bind).collect{|m| m.to_s}
-        iv = eval("instance_variables", bind).collect{|m| m.to_s}
+        gv = eval("send(:global_variables)", bind).collect{|m| m.to_s}.push("true", "false", "nil")
+        lv = eval("send(:local_variables)", bind).collect{|m| m.to_s}
```

Comment:
Under Ruby 2.6, private methods cannot be called with `self.`, so `#send` is used.